composer.json

```diff
@@ -19,7 +19,8 @@
 	"require-dev": {
 		"phpunit/phpunit": "^8.0 || ^9.0",
 		"mikey179/vfsstream": "^1.2",
-		"vimeo/psalm": "^4.8"
+		"vimeo/psalm": "^4.8",
+		"microsoft/microsoft-graph": "^1.34"

```

Comment:
This is used to make the tests involving the models pass for now by adding the current package as a dev dependency.

Reply:
I don't think we should have functional tests or model tests in this package. Core's tests should test Core's functionality. Makes more sense to me have functional and model tests in beta and v1 packages since they have Core as a dependency and hence can do end-to-end tests.